src/Ocelot.Provider.Consul/Consul.cs

```diff
@@ -49,6 +51,8 @@ public async Task<List<Service>> Get()
                         var serviceNode = nodes.Response.FirstOrDefault(n => n.Address == serviceEntry.Service.Address);
                         services.Add(BuildService(serviceEntry, serviceNode));
                     }
+
+                    _logger.LogDebug($"Consul answer: Address: {serviceEntry.Service.Address}, Port: {serviceEntry.Service.Port}");
```

Comment:
Long expressions of property reading with multiple references.
```suggestion
                    var address = serviceEntry.Service.Address;
                    var port = serviceEntry.Service.Port;
                    _logger.LogDebug($"Consul answer: Address: {address}, Port: {port}");
```

Reply:
✔️ Fixed in https://github.com/ThreeMammals/Ocelot/pull/1366/commits/5a5f1c01949e74494aa3e03bb5a9d99fce4b95ce